src/appkit.cs

```diff
@@ -744,6 +746,10 @@ public interface NSApplication : NSAccessibilityElementProtocol, NSAccessibility
 		[Export ("orderFrontStandardAboutPanelWithOptions:")]
 		void OrderFrontStandardAboutPanelWithOptions (NSDictionary optionsDictionary);
 #endif
+
+		[Mac (10,12)]
+		[Export ("enumerateWindowsWithOptions:usingBlock:")]
+		unsafe void EnumerateWindows (NSWindowListOptions options, Action<NSWindow, /*bool* */ IntPtr> block);
```

Comment:
Yeah. I was going to ask how to handle this in review. I tried an inline Action <NSWindow, ref bool> and that failed. Let me try a named delegate.